test/system/src/test/java/io/pravega/test/system/AbstractFailoverTests.java

```diff
@@ -144,6 +144,57 @@ void performFailoverForTestsInvolvingTxns() throws ExecutionException {
                 testState.getEventReadCount(), testState.getEventWrittenCount());
     }
 
+    void performFailoverForTestsInvolvingKVS() throws ExecutionException {
+        long kvpEntryCount = getListKvp();
+        log.info("Current insert KVP count {}", kvpEntryCount);
+        log.info("ScaleUp one Segmentstore and Controller");
+
+        // Case-64 Scale up SegmentStore one instances from 3 to 4
```

Comment:
This is open source code, `Case-XX` does not mean anything here. Please delete all these references.

Reply:
Resolved, Removed case-XX from comment